selfdrive/manager/process_config.py

```diff
@@ -35,8 +38,8 @@ def qcomgps(started, params, CP: car.CarParams) -> bool:
   NativeProcess("clocksd", "system/clocksd", ["./clocksd"]),
   NativeProcess("logcatd", "system/logcatd", ["./logcatd"]),
   NativeProcess("proclogd", "system/proclogd", ["./proclogd"]),
-  PythonProcess("logmessaged", "system.logmessaged", offroad=True),
-  PythonProcess("micd", "system.micd"),
+  PythonProcess("logmessaged", "system.logmessaged", offroad=True, callback=iscar),
+  PythonProcess("micd", "system.micd", callback=iscar),
```

Comment:
same here

Reply:
@haraschax Was this because we want microphone control outside of `micd`?

Reply:
yes